homepage.css

```diff
@@ -0,0 +1,444 @@
+    color: #aaa;
+}
+
+.icons img{
+    width: 20%;
+    height:85%;
+    cursor: pointer;
+    margin-left: 15px;
+    border-radius: 50%;
+}
+
+.icons i {
+  margin-left: 15px;
+}
+
+
+main{
+  display: flex;
+}
+
+.iframe-container{
+  position: relative;
+  width: 100%;
+  padding-bottom: 56.25%;
+  height: 0;
+}
+.iframe-container iframe{
+  position: absolute;
+  top:0;
+  left: 0;
+  width: 98.7%;
+  height: 100%;
+}
+
+.stats {
+  display: flex;
+  align-items: flex-end;
+}
+
+.views-number {
+  width: 40%;
+  height: 100%;
+}
+
+.views-number p {
+  font-size: 19px;
+  margin-bottom: 2px;
+  color: rgb(102, 102, 102);
+}
+
+#views-hr {
+  margin-top: 1px;
+  margin-left: -48px;
+  width: 98%;
+  margin-bottom: -3px;
+  border: 1px solid rgb(32, 124, 196);
+}
+
+.title-stats {
+  background-color: #fff;
+  width: 98.5%;
+  padding-left: 20px;
+  margin-top: 5px;
+  padding-top: 5px;
+}
+
+.title-stats h2{
+  font-size: 20px;
+}
+
+.video-container{
+  width: 60%;
+  margin-left: 30px;
+}
+
+aside{
+  width: 35%;
+  background-color: #FFF;
+  height:100%;
+}
+
+
+.preview-container{
+  display: flex;
+}
+
+.control-recommended-video-list{
+  display: flex;
+  justify-content: space-between;
+}
+.right-most{
+  display: flex;
+  align-items: center;
+  justify-content:space-evenly;
+
+}
+
+.right-most img{
+  width: 20%;
+  margin-left: -3px;
+  cursor: pointer;
+}
+
+.right-most p{
+  color: rgb(118, 118, 118);
+}
+
+.preview-channel-name, .preview-views{
+  color: rgb(118, 118, 118);
+  font-size: 12px;
+  margin-top: -9px;
+}
+
+
+
+.fa-toggle-on{
+  color: rgb(32, 124, 196);
+  font-size: 28px;
+  margin-right: 15px;
+  cursor: pointer;
+}
+
+.left-most p{
+  margin-left: 30px;
+}
+
+.thumbnail{
+  width: 65%;
+  margin-bottom: 11px;
+  margin-right: 10px;
+}
+
+.thumbnail img{
+  width: 100%;
+  height: 100%;
+  margin-left: 15px;
+  margin-right: 15px;
+}
+
+.preview-videoTitle {
+  margin-top: 0;
+  font-weight: 300;
+  color: rgb(51, 51, 51);
+
+}
+
+ul{
+  list-style: none;
+  margin-left:-7px;
+}
+
+#previewContainer-hLine, #previewContainer-hLine2{
+    border: none;
+    height: 1px;
+    color:rgb(226, 226, 226) ;
+    background-color: rgb(226, 226, 226) ;
+    margin-bottom: 20px;
+    margin-top: -10px;
+    margin-left: -11px;
+    margin-right: 15px;
+}
+.preview-info{
+  width: 70%;
+  margin-left: 30px;
+  margin-right: 15px;
+}
+
+.preview-info a{
+  text-decoration: none;
+}
+
+
+.preview-showmore-container{
+text-align: center;
+padding-bottom: 10px;
+}
+.preview-showmore-container a {
+  text-decoration: none;
+  color: rgb(118, 118, 118);
+}
+
+.preview-showmore-container ul {
+  margin-bottom: -10px;
+}
+
+
+.sNumber{
+  display: flex;
+  align-items: center;
+}
+
+#subscribers-number {
+  border: 1px solid rgb(209, 209, 209);
+  background-color: rgb(250, 250, 250);
+  color: rgb(115, 115, 115);
+  font-size: 11px;
+  padding: 4px 7px;
+}
+
+.subscribe{
+  /*display: flex;
+  justify-content: space-evenly;*/
+}
```

Comment:
Comments: It’s better to delete commented piece of codes. If you need to review a previous version of your code, use git history (screenshot from Pathwright)
❌ **commented piece of codes** `/* .nav { color: red;} */`
✔ **comments about code** `/*Navigation */`
Comments about the code is okay and should only be tagged as OPTIONAL if it's unnecessary and please provide details why it is unnecessary.